nuxeo-web-ui-ftest/test/features/step_definitions/recents.js

```diff
@@ -1,6 +1,28 @@
 'use strict';
 
 module.exports = function () {
-  this.Then('I can see the list of recently viewed documents',
-      () => this.ui.drawer.recents.waitForVisible().should.be.true);
+  this.Given(/^I have a clean recently viewed documents list/, () => {
+    fixtures.documents.clearLocalStorage(this.username, 'nuxeo-recent-documents');
+    fixtures.documents.reloadLocalStorage('* >>> nuxeo-recent-documents >>> #storage');
```

Comment:
We usually do not expose selectors with the shadow piercer in the step definitions, since this is ver specific to how tests hard being handled with shadow dom. Usually we hide these details on fixture objects or page helpers. Since this applies to recents and you're already adding a page helper for it, we should be doing something like `this.ui.drawer.recents.reload()` instead.